scripts/globals/items.lua

```diff
@@ -4396,6 +4457,10 @@ xi.items =
     FAUCHEUSE                       = 18952,
     GLEAMING_ZAGHNAL                = 18963,
     DIRE_SCYTHE                     = 18964,
+    TERPSICHORE                     = 18989,
```

Comment:
There are 11 weapons called this. I think we have a special naming convention for this

Reply:
Can you rename it to `TERPSICHORE_75` ?

We will have to rename other relics and mythics to align with this, but this is out of the scope of this PR

Reply:
> Can you rename it to `TERPSICHORE_75` ?
> 
> We will have to rename other relics and mythics to align with this, but that is out of the scope of this PR

updated to reflect the level